paywall/src/services/walletService.js

```diff
@@ -139,7 +140,7 @@ export default class WalletService extends EventEmitter {
    * receipt... etc)
    * @private
    */
-  _sendTransaction({ to, from, data, value, gas }, callback) {
+  _sendTransaction({ to, from, data, value, gas }, callback, transactionType) {
```

Comment:
Please always use the callback as the _last_ argument on a function...

Reply:
Ok. Another thought: should I put the translation type inside the object we pass instead of passing a new parameter?

Reply:
I'd say no because I think this type is really specific to Unlock while the other fields are generic in Ethereum.

Reply:
Sounds good.